### ISSUE TITLE
Bug: $getObjectProperty is not read if the key is a number

### ISSUE THREAD

**Issue by MuhammedEp** [aoijs/aoi.js#564]
### Bug Type

Function

### Bugged Function

```js
modüle.exports={
name:"test",
code:`
$getObjectProperty[json;$authorID]
$createObject[json;$readFile[example.json]]
`
}

//  JSON FİLE
{
"858951074164244490":"Example-text",
"Test":"Hi"
}
```

### Version of aoi.js

v6.5.6

### Version of Node.js

v16.11.0 (Lowest)

### Excepted Output

As the author of the command, I want to output "example-text" (without quotes) with authorid

### Relevant log output

```shell
$getObjectProperty[json;$authorID]
returns "undefined"
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Faf4a** [comment]
Thank you for reporting this bug, I can indeed confirm it!

**Comment by Faf4a** [comment]
Fixed in [598a24f](https://github.com/AkaruiDevelopment/aoi.js/commit/598a24f4dc651f683e14aad89ef6299d0aff6f51)
